clients/spring-boot-starter-camunda-sdk-deprecated/pom.xml

```
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.camunda</groupId>
+    <artifactId>zeebe-parent</artifactId>
+    <version>8.8.0-SNAPSHOT</version>
+    <relativePath>../../parent/pom.xml</relativePath>
+  </parent>
+  <artifactId>spring-boot-starter-camunda-sdk</artifactId>
+  <version>8.8.0-SNAPSHOT</version>
+  <name>Camunda Spring Boot Starter (Deprecated)</name>
```

🔧  I would suggest to also use the old name here
```suggestion
  <name>Camunda Spring SDK (Deprecated)</name>
```

---

@copilot can you address this, please?

---

Your change is not what I suggested, are you actually able to read the content of the suggested change? Or did you just act on my ambiguous wording?

---

@copilot please answer my previous message.

---

In the meantime, please change the name of the deprecated module to "Camunda Spring SDK (Deprecated)" and we are good :)